Encounter1/MenuActivity.cs

```diff
@@ -14,12 +14,20 @@ namespace Encounter1
     [Activity(Label = "MenuActivity")]
     public class MenuActivity : Activity
     {
+        Button btnLocationList;
         protected override void OnCreate(Bundle bundle)
         {
             base.OnCreate(bundle);
             SetContentView(Resource.Layout.Menu);
             TextView currentCharacterName = FindViewById<TextView>(Resource.Id.textViewUserName);
             currentCharacterName.Text = Intent.Extras.GetString("userName");
+            btnLocationList = FindViewById<Button>(Resource.Id.button1);
+            btnLocationList.Click += OnLocationListButtonClicked;
```

Comment:
maybe we should start grouping objects in methods? For example buttons in ButtonInitialization() so the OnCreate() method is less cluttered

Reply:
I think we will definitely have to do that later on, however, now it would only change two lines to one and add more complexity with additional function.
On unrelated note, in another commit it might be a good idea to move most of interactions with database to a separate class.